### DISCUSSION TITLE
How do I change the location of the config file for "Open Config File" under "Utils"?

### DISCUSSION THREAD

**Discussion by refugeindhamma** [tomasklaen/uosc#369] [Q&A]
Hi, I would like to change the location of the config file for "Open Config File" under "Utils" but have no idea how to do it. I am using Windows 11 and the mpv.conf file is located at C:\Users\Public\Videos\MPV.io\MPV Player\script-opts. What is the command to enter into the mpv.conf file?

**Comment by tomasklaen** [comment]
The `open-config-directory` command simply asks mpv where is it reading the config file from and opens that. This is not configurable.

**Reply by refugeindhamma** [reply]
> The `open-config-directory` command simply asks mpv where is it reading the config file from and opens that. This is not configurable.

MPV is able to read my mpv.conf file from the directory stated above but why is it that it does not open the directory from the Utils? It opens the Desktop instead.

**Reply by tomasklaen** [reply]
Oh, that's #365 issue, that has been fixed but not released yet. You can copy the `scripts/uosc.lua` file from main branch.

**Comment by maaldheefee** [comment]
`# run explorer C:\Users\Public\Videos\MPV.io\MPV Player\script-opts #! Utils > Open config directory`

**Reply by refugeindhamma** [reply]
Sorry I'm lost. Do I put this code in mpv.conf or uosc.conf? Also, wouldn't "#" comment it out of the code?

**Reply by FichteFoll** [reply]
`input.conf`, see https://github.com/tomasklaen/uosc/tree/b75ceb6d6fdd37b08b1c3c967fcc009711541b76#adding-items-to-menu for documentation.
